assets/css/media.css

```diff
@@ -0,0 +1,343 @@
+@media (min-width: 765px) {
```

Comment:
The `min-width` should be set to `768px`

Reply:
Thanks for the suggestion we set the min-width to 768px